chapters/ch14/4.question-answering-demo-application.ipynb

```diff
@@ -0,0 +1,475 @@
+       "      <td>What are minimalist shoes?</td>\n",
+       "      <td>One summer job, I needed shoes to walk on a ro...</td>\n",
+       "      <td>https://outdoors.stackexchange.com/questions/1...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>18375</td>\n",
+       "      <td>What are minimalist shoes?</td>\n",
+       "      <td>The answer to this question will vary on your ...</td>\n",
+       "      <td>https://outdoors.stackexchange.com/questions/1...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>13540</td>\n",
+       "      <td>What are minimalist shoes?</td>\n",
+       "      <td>Barefoot Shoes Also known as minimalist shoes,...</td>\n",
+       "      <td>https://outdoors.stackexchange.com/questions/1...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      id                    question  \\\n",
+       "0  18376  What are minimalist shoes?   \n",
+       "1  18370  What are minimalist shoes?   \n",
+       "2  16427  What are minimalist shoes?   \n",
+       "3  18375  What are minimalist shoes?   \n",
+       "4  13540  What are minimalist shoes?   \n",
+       "\n",
+       "                                             context  \\\n",
+       "0  Minimalist shoes or \"barefoot\" shoes are shoes...   \n",
+       "1  There was actually a project done on the defin...   \n",
+       "2  One summer job, I needed shoes to walk on a ro...   \n",
+       "3  The answer to this question will vary on your ...   \n",
+       "4  Barefoot Shoes Also known as minimalist shoes,...   \n",
+       "\n",
+       "                                                 url  \n",
+       "0  https://outdoors.stackexchange.com/questions/1...  \n",
+       "1  https://outdoors.stackexchange.com/questions/1...  \n",
+       "2  https://outdoors.stackexchange.com/questions/1...  \n",
+       "3  https://outdoors.stackexchange.com/questions/1...  \n",
+       "4  https://outdoors.stackexchange.com/questions/1...  "
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "example_contexts = retriever('What are minimalist shoes?')\n",
+    "example_contexts[0:10]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Listing 14.16"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "OSError",
+     "evalue": "Can't load the configuration of '../../data/roberta-base-squad2-outdoors'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure '../../data/roberta-base-squad2-outdoors' is the correct path to a directory containing a config.json file",
```

Comment:
@dcrouch26 output has errors. Please rerun to ensure these are working and their outputs are consistent with the book.